ST_structs.h

```diff
@@ -1,333 +1,721 @@
-  IntUS *kills,        /* ptr to array of # indivs killed by age. index=age. */
-        estabs,        /* number of individuals established in iter */
-        *seedprod,    /* annuals: array of previous years' seed production (size = viable_yrs)*/
+      /** \brief Number of individuals established (growing) */
+  SppIndex est_count;
+      /** \brief Pointer to array of individuals killed, indexed by age at death. */
+  IntUS *kills,
+      /** \brief Number of individuals established in current iteration. */
+        estabs,
+      /** \brief Specific to annuals. Array of seeds produced indexed by year. 
+       * \sa ST_seedDispersal.c */
+        *seedprod,
+      /** \brief not sure what this does yet.
+       * \sa ST_seedDispersal.c */
          seedbank,
-         pseed;       /* average number of seeds produced by annual species per 1g of biomass, per 1m^2 and per year (internally re-calculated as seeds per 1 g biomass per plot and per year) */
-  RealF lastyear_relsize,    /* relsize from the previous year, used for annual establishment */
-        extragrowth,   /* amt of superfluous growth from extra resources */
-	received_prob,	//the chance that this species received seeds this year... only applicable if using seed dispersal and gridded option
-        alpha,         /* species-specific alpha parameter used for random number draw from beta distribution in establishment of annual species */
-        beta;           /* species-specific beta parameter used for random number draw from beta distribution in establishment of annual species */
-  float       var;    /* the variance parameter of the beta distribution for establishment..*/
-  struct indiv_st *IndvHead;    /* facility for linked list 8/3/01; top of list */
-  Bool allow_growth, //whether to allow growth this year... only applicable if using seed dispersal and gridded option
-	sd_sgerm;	//whether seeds where produced/received and germinated this year... only applicable if using seed dispersal and gridded option
+      /** \brief Average number of seeds produced by annual species per 1g of biomass, per 1m^2 and per year.
+       * internally re-calculated as seeds per 1 g biomass per plot and per year. */
+         pseed;
+      /** \brief relsize from the previous year, used for annual establishment. 
+       * \sa rgroup_Establish() */
+  RealF lastyear_relsize,
+      /** \brief Amount of superfluous growth from extra resources. */
+        extragrowth,
+      /** \brief Probability that this species received seeds this year. */
+	      received_prob,
+      /** \brief Alpha parameter for random number draw from beta distribution in establishment of annual species. 
+       * \sa _add_annuals() */
+        alpha,
+      /** \brief Beta parameter for random number draw from beta distribution in establishment of annual species.
+       * \sa _add_annuals() */
+        beta;
+      /** \brief Variance parameter of the beta distribution for establishment. */
```

Comment:
change to " for establishment of annual species"

Reply:
Addressed by commit 984bd13.